src/data/screenshots_de.json

```diff
@@ -5,17 +5,14 @@
                 "Mit diesem Update stellen wir neue und erweiterte Funktionen zur Verfügung:",
-                "<b>Hinweise bei positivem Test:</b> Die Hinweise und Vorgehensweise bei einem positiven Testergebnis wurden überarbeitet.",
-                "<b>Statistik zu Auffrischimpfungen:</b> Neben den Statistiken für mindestens einmal geimpfte und vollständig geimpfte Personen, finden Sie jetzt auch die Anzahl der Personen, die eine Auffrischimpfung erhalten haben.",
-                "<b>Link zu Social Media:</b> Wenn Sie über die Corona-Warn-App auf dem Laufenden bleiben und sich mit anderen austauschen wollen, können Sie unseren Social-Media-Kanälen folgen. Android-Nutzer*innen finden den Link zu den Social-Media-Kanälen, indem sie auf die drei Punkte in der oberen Rechten Ecke der App tippen und dann „Social Media“ auswählen. iOS-Nutzer*innen können in der CWA nach unten wischen und dort unter dem Punkt „Mehr“ „Social Media“ auswählen.",
-                "<b>Löschdatum für Papierkorbinhalte</b>: Jedes Element, dass in den Papierkorb verschoben wird, wird nach 30 Tagen endgültig gelöscht. Dieses Löschdatum wird jetzt für jedes Element angezeigt.",
-                "<b>Nachricht bei erhöhtem Risiko</b>: Sie erhalten trotz geöffneter App eine Benachrichtigung, wenn ein erhöhtes Infektionsrisikio für Sie ermittelt wird. Damit soll vermieden werden, dass Sie die Risiko-Warnung auf der Status-Registerkarte übersehen, falls Sie sich gerade auf einer anderen Registerkarte befinden. Zusätzlich wird auf der Status-Registerkarte ein roter Punkt erscheinen, solange Sie diese noch nicht angeschaut haben. Diese Funktion wurde bereits in Version 2.16 für iOS hinzugefügt und ist mit dieser Version nun auch für Android verfügbar.",
+                "<b>Zentrale Verwaltung von Regeln:</b> Die Regeln für Zertifikate (z.&nbsp;B. hinsichtlich ihrer Gültigkeit, oder Priorisierung, wenn mehrere Zertifikate vorhanden sind) und für die Ermittlung des G-Status wurden aus der Corona-Warn-App herausgelöst. Diese Regeln sind jetzt zentral abgelegt und können daher unabhängig von den App-Releases angepasst werden. Damit kann schneller auf Regeländerungen reagiert werden. Nach einer Änderung werden die aktualisierten Regeln an die App übertragen. Für Sie ändert sich an der Benutzung der Corona-Warn-App nichts durch diese zentrale Verwaltung der Regeln.",
+                "<b>Anpassung der Darstellung für Auffrischungsimpfung:</b> Im Rahmen der Erinnerung für Auffrischungsimpfungen gibt es nun eine übersichtliche Darstellung. Weitere Informationen zu einer empfohlenen Auffrischungsimpfung sind nun durch das Tippen auf das Feld “Hinweis zur Auffrischimpfung” einsehbar. Siehe Screenshots 'Hinweis zur Auffrischimpfung' für sowohl <a href='#android_booster_note' target='_blank'>Android</a> als auch <a href='#ios_booster_note' target='_blank'>iOS</a> ",
```

Comment:
Auffrischungsimpfung => Auffrischimpfung

See issue https://github.com/corona-warn-app/cwa-website/issues/2162 from @svengabr

Reply:
Fixed it in commit [8eb7705](https://github.com/corona-warn-app/cwa-website/pull/2452/commits/8eb7705727bcb144955f83c075361327834dd115), thanks!